### ISSUE TITLE
Failed to find catalog by ID

### ISSUE THREAD

**Issue by JSimon000** [rancher/rancher#20034]
**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
When upgrading from 2.2.3-rc6 to 2.2.3-rc7 rancher is unable to find the monitoring templates

**Result:**
an error is thrown indicating the failure to find catalog by ID

**Other details that may be helpful:**

**Environment information**
Rancher version 2.2.3rc7 HA install

```
```

  | Failed to ensure catalog "catalog://?catalog=system-library&template=rancher-monitoring&version=0.0.3": failed to find catalog by ID "catalog://?catalog=system-library&template=rancher-monitoring&version=0.0.3": catalogtemplateversions.management.cattle.io "system-library-rancher-monitoring-0.0.3" not found
-- | --

**Comment by JSimon000** [comment]
I noticed the chart is only updated in the dev branch of the charts so I added that catalog to the deployment but it still fails to find the catalog. I am not sure if this is all expected given the rc status of the build. If so I do apologize for the spurious issue.

If not this is the exception after I disable monitoring and add the catalog and also if I try to disabling the default catalog.

`Catalogtemplateversions.management.cattle.io "system-library-rancher-monitoring-0.0.3" not found
`
